hack/deploy_csv.yaml

```diff
@@ -250,7 +250,8 @@ data:
           * Red Hat CodeReady Workspaces
           * OpenShift Pipelines
           ## Installation
-          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and the developer tools get installed in a single namespace.
+          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-che` namespace.
```

Comment:
```suggestion
          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-workspaces` namespace.
```

Reply:
@alexeykazakov I'm afraid we still use the `toolchain-che` project...

Reply:
let me open a PR to change that

Reply:
@alexeykazakov sorry, but I missed your comments/suggestions - I changed it here: d58c62c